### PR TITLE
Added Group Details Screen.

### PR DESCRIPTION
**Pull request by vishwesh3**
Which is shown on touching Group Title.

this PR is against #312 

@borisyankov @nashvail I have used cards to show members list. What other UX you are thinking of?

screen shot 2017-07-07 at 6 36 40 pm
screen shot 2017-07-07 at 6 36 51 pm

### CODE REVIEW

**Comment by borisyankov** [comment]
Hmmm.... One thing we might prefer is to have larger user icons, so the list items might be more square. Don't feel too strongly about this.

Another thing is... we might have more of a sole screen dedicated to that, depending on if we need to show more information (or if we can show)

Let this PR sit like this for a while and let's consider what other UI patterns are we missing. Please, do play a lot with several of the big competitors' apps and let's discuss UI screenshots?

File: src/account-info/AccountDetailsScreen.js [hunk 1]
```diff
@@ -28,7 +27,8 @@ class AccountDetailsScreen extends Component {
   componentWillMount() {
     // props.email gets reset during navigation slide out (on back)
     // so we cache value to prevent an exception
-    this.user = this.props.users.find(x => x.email === this.props.email) || NOT_FOUND_USER;
+    const { users, email } = this.props;
+    this.user = users.find(x => x.email === email) || NOT_FOUND_USER;
```
Comment (borisyankov) [inline src/account-info/AccountDetailsScreen.js]: This will be better served as a selector.

File: src/i18n/translations/messages_en.json [hunk 2]
```diff
@@ -57,5 +57,7 @@
   "Jot down something": "Jot down something",
   "Message {recipient}": "Message {recipient}",
   "Send private message": "Send private message",
-  "Forgot password?": "Forgot password?"
+  "Forgot password?": "Forgot password?",
+  "Members": "Members",
+  "Group Details": "Group Details",
```
Comment (borisyankov) [inline src/i18n/translations/messages_en.json]: The 'D' in details should be lower case.

File: src/account-info/AccountDetailsScreen.js (hunk 1 above)
Comment (vishwesh3) [inline src/account-info/AccountDetailsScreen.js]: Done :+1:

File: src/i18n/translations/messages_en.json (hunk 2 above)
Comment (vishwesh3) [inline src/i18n/translations/messages_en.json]: Done :+1:

**Comment by vishwesh3** [comment]
@borisyankov Updated. Now it works with React Navigation 👍

**Comment by borisyankov** [comment]
I PR I merged broke it again this time the fix should be quick and minor 👍

**Comment by vishwesh3** [comment]
@borisyankov Done 👍 :)

**Pull request merged**